src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -611,6 +611,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <ContinueOnError Condition="'$(ContinueOnError)' == ''">false</ContinueOnError>
   </PropertyGroup>
 
+  <!-- Users familiar with how some other repos work try to use NoWarn with MSBuild in place of MSBuildWarningsAsMessages. -->
+  <PropertyGroup Condition="'$(MSBuildWarningsAsMessages)'=='' And $(MSBuildChangeWaveVersion) != '' And $([MSBuild]::VersionGreaterThanOrEquals('$(MSBuildChangeWaveVersion)', '16.8'))">
```

```suggestion
  <PropertyGroup Condition="'$(MSBuildWarningsAsMessages)'=='' and $(MSBuildChangeWaveVersion) != '' and $([MSBuild]::VersionLessThan('16.8', '$(MSBuildChangeWaveVersion)'))">
```
Nit: The dev docs will suggest comparing versions in this way. In my opinion it reads clearer.

---

Who wrote the dev docs? I don't agree.

---

It'll be added to the wiki as a markdown file. It's preference at that point 👍

---

Also, if you're trying to enable this so long as the change wave version is greater than 16.8, you don't want to use `VersionGreaterThanOrEquals`, as setting change wave `16.8` is intended to disable any features behind that.

---

We should talk about this in the PR meeting.

---

This should now be:
```suggestion
  <PropertyGroup Condition="'$(MSBuildWarningsAsMessages)'=='' And $([MSBuild]::IsChangeWaveEnabled('16.8'))">
```

The intrinsic function returns false on empty `MSBuildChangeWaveVersion` values.